src/controllers/DappsStakingController.ts

```diff
@@ -334,6 +336,41 @@ export class DappsStakingController extends ControllerBase implements IControlle
             },
         );
 
+        app.route('/api/v3/:network/dapps-staking/stats/user/:userAddress/:period').get(
+            async (req: Request, res: Response) => {
+                // this._giantSquidService.getUserCalls(req.params.network as NetworkType, req.params.userAddress, req.params.period as PeriodType);
```

Comment:
Few things update here:
- remove commented code
- add an endpoint description for Swagger
- we need to fetch events by `contractAddress`, not `userAddress`
- think about idea to have a date as parameter instead of range. Most likely we will need events for one day. Also, in case of period we need to add paging to the endpoint, because there will be a lot of records